lib/storage/metricnamestats/tracker.go

```diff
@@ -74,19 +74,20 @@ type recordForStore struct {
 // MustLoadFrom inits tracker from the given on-disk path
 func MustLoadFrom(loadPath string, maxSizeBytes uint64) *Tracker {
 	mt, err := loadFrom(loadPath, maxSizeBytes)
-	if err != nil {
-		logger.Fatalf("unexpected error at tracker state load from path=%q: %s", loadPath, err)
+	if err == nil {
+		return mt
 	}
-	return mt
+	if errors.Is(err, os.ErrNotExist) {
+		logger.Infof("tracker store file at path %s missing files; init new tracker", loadPath)
```

Comment:
I would add context here and hereafter about what is `tracker`, because users might not be able to understand what's failing.
Maybe use `metric names tracker` instead?